scripts/release

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env ruby
+
+require "date"
+require "optparse"
+
+def run(command, abort_on_fail: true)
+  puts "> #{command}" if $VERBOSE
+
+  system(command, exception: abort_on_fail)
```

Comment:
Nice, I totally forgot about the new-ish `exception:` argument 👏 No more `fail unless $?.existstatus.zero?` hacks 😃

Reply:
lol I actually discovered this while looking up the return value for `system` so I could raise the exception. :joy: It's possible I don't need to rescue and reraise here, but I wanted to make sure to include the command in the exception and this does that without someone reading the code having to guess at it.

I would've used Bash for these scripts so I could just use `set -e` in the script for this exact functionality, but I needed to parse CLI options and I have no idea how to do that in Bash. 😄

Reply:
I think Ruby is fine for this script but re argument parsing in Bash, you might be interested in https://argbash.io.